docs/source/markdown/podmansh.1.md

```diff
@@ -52,13 +54,19 @@ RequiredBy=default.target
 _EOF
 ```
 
-Users inside of this Quadlet are allowed to become root within the user namespace, and able to read/write content in their homedirectory which is mounted from a subdir `data` of the hosts users account.
+Alternatively, while runnig as root, create a Quadlet where users are allowed to become root within the user namespace, and able to read/write content in their homedirectory which is mounted from a subdir `data` of the hosts users account.
```

Comment:
Typo, missing 'n' in 'runnig'. And I think it would read better as:
>, and **ARE** able to read/write ...

Reply:
Maybe "Host's user account(s)"?